plans/0.5.0-operator-experience.md

```
@@ -0,0 +1,157 @@
+Status: proposed
+Owner: Shipper swarm maintainers
+Created: 2026-08-20
+Milestone: 0.5.0 operator-experience closeout
+Linked proposal: docs/proposals/SHIPPER-PROP-0001-source-of-truth-and-release-evidence.md
+Linked specs: docs/specs/SHIPPER-SPEC-0004-json-evidence-contracts.md;
+docs/INVARIANTS.md
+Linked ADRs: docs/adr/SHIPPER-ADR-0002-registry-truth-over-cargo-output.md
+Linked plan: plans/0.5.0-operator-experience.md
+Linked issues: #268; #269; #270; #271; #272; #274; #275
+Linked PRs: #278; #279; #282; #295; #308; #309; #311; #314; #315;
+#317; #318; #320; #325; #326
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)